slsSupportLib/include/sls/sls_detector_defs.h

```diff
@@ -160,6 +176,36 @@ class slsDetectorDefs {
         sls_detector_header detHeader; /**< is the detector header */
         sls_bitset packetsMask;        /**< is the packets caught bit mask */
     };
+
+    struct startCallbackHeader {
+        std::vector<uint32_t> udpPort;
```

Comment:
How many ports can we have?

Reply:
1 or 2 udp ports per startcallback and endcallbackheader

Reply:
I'm torn if we should keep a vector that has one or two elements or if we should use our own StaticVector to avoid heap allocations.